tst/CTA.Rules.Test/TempRules/microsoft.owin.hosting.json

```diff
@@ -115,7 +115,8 @@
               "Description": "Add a comment to explain how to replace WebApp.Start.",
               "ActionValidation": {
                 "Contains": "ReplaceMicrosoft.Owin.Hosting.WebApp.StartwithWebHostBuilder",
-                "NotContains": ""
+                "NotContains": "",
+                "CheckComments":  "True" 
```

Comment:
If changing `CheckComments` to bool, need to update value to `true` (this applies to other occurrences below).

Reply:
Mark also mentioned he would rather keep everything as strings for now and not introduce new data types currently.